pragma-sdk/pragma_sdk/common/types/merkle_tree.py

```diff
@@ -0,0 +1,51 @@
+from typing import List, Any
+from starknet_py.utils.merkle_tree import MerkleTree as StarknetPyMT
+
+
+MerkleProof = List[int]
+
+
+class MerkleTree(StarknetPyMT):
```

Comment:
we really need some unit tests for this

Reply:
Removed this implementation - not needed